packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_CoarseSpace_def.hpp

```diff
@@ -134,17 +184,17 @@ namespace FROSch {
             for (UN j=0; j<AssembledBasis_->getNumVectors(); j++) {
                 valueTmp=AssembledBasis_->getData(j)[i];
```

Comment:
Constantly calling `getData` in an inner loop can be expensive.  Consider getting the data outside the loop.

Reply:
Thank you. I changed this.